web/template/mlpa-documents.gohtml

```diff
@@ -0,0 +1,64 @@
+{{ template "page" . }}
+
+{{ define "title" }}Documents{{ end }}
+
+{{ define "main" }}
+    <div class="govuk-grid-row" id="documents">
+        <div class="govuk-grid-column-two-thirds">
+
+            {{ template "mlpa-header" (caseTabs .Lpa "documents") }}
+
+            <div class="govuk-!-display-block">
+                <h1 class="govuk-heading-l govuk-!-margin-bottom-5 govuk-!-display-inline govuk-!-padding-right-4">Documents</h1>
+                <button class="govuk-button govuk-button--secondary" data-module="govuk-button">Create a document</button>
+            </div>
```

Comment:
There's an [MOJ component for this](https://design-patterns.service.justice.gov.uk/components/page-header-actions/) that might be safer than managing the padding manually